src/server/visual-diff-reporter.js

```diff
@@ -0,0 +1,74 @@
+import { basename, dirname, join } from 'path';
+import { readdirSync, rmSync } from 'node:fs';
+
+function getDiffPaths(dir, _diffPaths) {
+	_diffPaths = _diffPaths || [];
+	const paths = readdirSync(dir, { withFileTypes: true });
+	paths.forEach(path => {
+		const full = join(dir, path.name);
+		const base = basename(path.name);
+
+		if (path.isDirectory()) {
+			if (base === 'pass' || base === 'fail') _diffPaths.push(full);
+			getDiffPaths(full, _diffPaths);
+		} else {
+			if (base === 'report.json') _diffPaths.push(full);
```

Comment:
Put this in as a placeholder, not sure yet if these are individual html files scattered around, one big file at the base, etc.

Reply:
Probably will be `report.html`?

Reply:
Yeah not sure where I got `json` from lol